src/targets/gpu/include/migraphx/gpu/hipblaslt.hpp

```diff
@@ -76,14 +76,17 @@ inline auto hipblaslt_invoke(F f, Ts... xs)
 }
 
 template <class F, class Pack, class... Ts>
```

Comment:
```suggestion
// Invoke a Hipblaslt call.  If used to validate a call, set fatal_error = false to prevent
// throwing an exception on failure.
template <class F, class Pack, class... Ts>
```

Reply:
Thanks for the suggestion, I have added the comment.